heron/executor/src/python/heron_executor.py

```diff
@@ -55,7 +57,7 @@ def print_usage():
       " <scheduler_classpath> <scheduler_port> <python_instance_binary>"
       " <metricscachemgr_classpath> <metricscachemgr_masterport> <metricscachemgr_statsport>"
       " <is_stateful> <ckptmgr_classpath> <ckptmgr_port> <stateful_config_file> "
-      " <healthmgr_mode> <healthmgr_classpath> <cpp_instance_binary>")
+      " <healthmgr_mode> <healthmgr_classpath> <cpp_instance_binary> <jvm_remote_debugger_ports>")
```

Comment:
do we need to add one more param in config/heron.aurora

Reply:
currently remote debugging is implemented for localscheduler and kubernetes scheduler.